letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -493,7 +493,10 @@ def prepare_server_https(self, port, temp=False):
         """
         if "ssl_module" not in self.parser.modules:
             logger.info("Loading mod_ssl into Apache Server")
-            self.enable_mod("ssl", temp)
+            if self.config.func.__name__ == "auth":
```

Comment:
Not a huge fan of this solution, but otherwise it creates additional work for the plugin and presents more ways for the plugin to break.

Note: If auth is ever changed... enable_mod will always be called with temp=False.  This is far less likely to cause real problems than if it was based off of the specific commands ```run``` and ```install```